android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -196,13 +198,54 @@ class XMTPModule : Module() {
             }
         }
 
-        AsyncFunction("loadMessages") { clientAddress: String, topics: List<String>, conversationIDs: List<String?>, limit: Int?, before: Long?, after: Long? ->
+        AsyncFunction("loadMessages") { clientAddress: String, topic: String, limit: Int?, before: Long?, after: Long? ->
             logV("loadMessages")
-            val client = clients[clientAddress] ?: throw XMTPException("No client")
+            val conversation =
+                findConversation(
+                    clientAddress = clientAddress,
+                    topic = topic,
+                ) ?: throw XMTPException("no conversation found for $topic")
             val beforeDate = if (before != null) Date(before) else null
             val afterDate = if (after != null) Date(after) else null
 
-            client.conversations.listBatchMessages(topics, limit, beforeDate, afterDate).map {
+            conversation.messages(limit = limit, before = beforeDate, after = afterDate)
+                .map {
+                    EncodedMessageWrapper.encode(it)
+                }
+        }
+
+        AsyncFunction("loadBatchMessages") { clientAddress: String, topics: List<String> ->
```

Comment:
`loadMessages` - lists all the messages via the envelopesList query inside of the SDKs

`loadBatchMessages` - lists all the messages for a batch of topics with given pagination parameters. 

These are two different methods in the sdks and should be here as well. Previously both messages, and batch messages called the batch method.